hw/ip/otbn/doc/_index.md

```diff
@@ -261,9 +261,11 @@ All RW CSRs are set to 0 when OTBN starts (when 1 is written to {{< regref "CMD.
       <td>R</td>
       <td>
         <strong>RND</strong>.
-        A random number.
+        An AIS31-compliant class PTG.3 random number with guaranteed entropy and forward and backward secrecy.
```

Comment:
I didn't realise that EDN is actually DRG.3 class.
In that case we can claim enhanced backward secrecy
```suggestion
        An AIS31-compliant class DRG.3 random number with guaranteed entropy, forward secrecy and enhanced backward secrecy.
```

Reply:
Hi Vladimir, it can actually be *either* DRG.3 or PTG.3 depending on the configuration of the port.   I would leave this line *unchanged* as our intention is to rely on the enhanced forward *and* backward secrecy that PTG.3 offers for the `RND` port.

Reply:
Ultimately, I'd like to state here whatever we claim in the security target -- and that's likely to be PTG.3 going forward AFAIK. I'll leave this as-is for now, as I think it reflects the current thinking best.